nmt_chainer/evaluation.py

```diff
@@ -185,8 +185,10 @@ def beam_search_translate(encdec, eos_idx, src_data, beam_width = 20, beam_pruni
                 rescored_translations.append((tr, sc + reverse_scores[num_t], attn))
             translations = rescored_translations
                     
-        if score_is_divided_by_length:
+        if post_score_length_normalization == 'simple':
             translations.sort(key = lambda x:x[1]/(len(x[0])+1), reverse = True)
+        elif post_score_length_normalization == 'google':
+            translations.sort(key = lambda x:x[1]/( pow((len(x[0])+6), length_normalization_strength) / pow(6, length_normalization_strength) ), reverse = True)
```

Comment:
I was in doubt here.  In the simple case, why do we divide by (len(x[0])+1) instead of len(x[0])?